auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationContextConfigurationClient.java

```diff
@@ -196,6 +198,44 @@ private static AuthenticationConfiguration initializeConfiguration(final URI uri
         return configuration;
     }
 
+    /**
+     * Get all SSL contexts configured for this authentication context.
+     * This method is not part of the public API.
+     *
+     * @param authenticationContext the authentication context to examine (must not be {@code null})
+     * @return List of all configured SSL context belonging to the provided authentication context
+     */
+    public List<SSLContext> getConfiguredSSLContexts(AuthenticationContext authenticationContext) throws GeneralSecurityException {
```

Comment:
@darranl @fjuma I did not realize before, but when I move the public class DynamicSSLContextSpiImpl from elytron client to dynamic-ssl module, then I can't use these methods without them being public. 

So we either add them as public methods and use a disclaimer saying it is not part of the public API (in javadoc/blog etc.), or use a reflection. I decided to do the former but  let me know if you think otherwise. Thanks.

Reply:
I think looking at the two methods we should just accept them in a new public API, I don't think this will affect our plans to include in Elytron 1.16 as the methods have a well defined scope.

Reply:
+1